src/main/java/ldbc/snb/datagen/util/ConfigParser.java

```diff
@@ -93,30 +93,32 @@ public static void printConfig(Map<String, String> conf) {
-        conf.put("ldbc.snb.datagen.generator.startYear", "2010");
-        conf.put("ldbc.snb.datagen.generator.numYears", "3");
-        conf.put("ldbc.snb.datagen.generator.numThreads", Integer.toString(1));
-        conf.put("ldbc.snb.datagen.generator.deltaTime", "10000");
-        conf.put("ldbc.snb.datagen.generator.distribution.degreeDistribution", "ldbc.snb.datagen.generator.distribution.FacebookDegreeDistribution");
-        conf.put("ldbc.snb.datagen.generator.knowsGenerator", "ldbc.snb.datagen.generator.generators.knowsgenerators.DistanceKnowsGenerator");
-        conf.put("ldbc.snb.datagen.generator.person.similarity", "ldbc.snb.datagen.entities.dynamic.person.similarity.GeoDistanceSimilarity");
-        conf.put("ldbc.snb.datagen.serializer.format","CsvBasic"); // CsvBasic, CsvMergeForeign, CsvComposite, CsvCompositeMergeForeign
-        conf.put("ldbc.snb.datagen.serializer.compressed", Boolean.toString(false));
-        conf.put("ldbc.snb.datagen.serializer.outputDir", "./");
-        conf.put("ldbc.snb.datagen.serializer.socialNetworkDir", "./social_network");
-        conf.put("ldbc.snb.datagen.serializer.hadoopDir", "./hadoop");
-        conf.put("ldbc.snb.datagen.serializer.endlineSeparator", Boolean.toString(false));
-        conf.put("ldbc.snb.datagen.serializer.dateFormatter", "ldbc.snb.datagen.util.formatter.StringDateFormatter");
-        conf.put("ldbc.snb.datagen.util.formatter.StringDateFormatter.dateTimeFormat", "yyyy-MM-dd'T'HH:mm:ss.SSS+00:00");
-        conf.put("ldbc.snb.datagen.util.formatter.StringDateFormatter.dateFormat", "yyyy-MM-dd");
-        conf.put("ldbc.snb.datagen.parametergenerator.python", "python");
-        conf.put("ldbc.snb.datagen.parametergenerator.parameters", "true");
-        conf.put("ldbc.snb.datagen.mode", "interactive"); // interactive, bi, graphalytics, rawdata
-        conf.put("ldbc.snb.datagen.mode.bi.deleteType", "simple"); // simple or smart
-        conf.put("ldbc.snb.datagen.mode.bi.batches", "1");
-        conf.put("ldbc.snb.datagen.mode.interactive.numUpdateStreams", "1");
+        conf.put("generator.numPersons", "10000");
+        conf.put("generator.startYear", "2010");
+        conf.put("generator.numYears", "3");
+        conf.put("generator.delta", "10000");
+        conf.put("generator.mode", "interactive");
+        conf.put("generator.mode.bi.deleteType", "simple");
+        conf.put("generator.mode.bi.batches", "1");
+        conf.put("generator.mode.interactive.numUpdateStreams", "1");
+        conf.put("generator.degreeDistribution", "Facebook");
+        conf.put("generator.knowsGenerator", "Distance");
+        conf.put("generator.person.similarity", "GeoDistance");
+        conf.put("generator.dateFormatter", "StringDate");
+        conf.put("generator.StringDate.dateTimeFormat", "yyyy-MM-dd'T'HH:mm:ss.SSS+00:00");
+        conf.put("generator.StringDate.dateFormat", "yyyy-MM-dd");
+
+
+        conf.put("hadoop.numThreads", "1");
```

Comment:
as this should be hadoop independent code, it would be even better if these config params were added somewhere else, maybe directly in `LdbcDatagen`